### ISSUE TITLE
[FirstContact] Stuck during the Antura Space customization tutorial

### ISSUE THREAD

**Issue by FrancyCavallari** [vgwb/Antura_arabic#670]
I'm not 100% sure about how I got it... 
... but I think double clicked the Category button and I got this:
![image](https://user-images.githubusercontent.com/17926293/35188434-9344f376-fe35-11e7-9eab-807cc83cc828.png)
It seems like the Tutorial progressed one extra step because a double clicked.

In order to reproduce it I decided to quit the game and restart...
... I got automatically back to the Antura Customization tutorial, but as soon as I click the Customization button I get this error and I am stuck:
`Exception: No new category!
Antura.AnturaSpace.AnturaSpaceTutorialManager.<StepTutorialCustomization>m__2 () (at Assets/_app/_scripts/AnturaSpace/AnturaSpaceTutorialManager.cs:209)
Antura.AnturaSpace.AnturaSpaceTutorialManager+<DelayedCallbackCO>c__Iterator0.MoveNext () (at Assets/_app/_scripts/AnturaSpace/AnturaSpaceTutorialManager.cs:590)
UnityEngine.SetupCoroutine.InvokeMoveNext (IEnumerator enumerator, IntPtr returnValueAddress) (at C:/buildslave/unity/build/Runtime/Export/Coroutines.cs:17)
`

**Comment by FrancyCavallari** [comment]
Since the profile becames unusable (it's always stuck in the Customization tutorial), better implement a fallback case: if there's no new category we use the first one (Head).

**Comment by FrancyCavallari** [comment]
Safaa is getting stuck 100% in the Customization Tutorial (she plays on Mac).
![image](https://user-images.githubusercontent.com/17926293/35201024-bf36beb4-ff17-11e7-9b34-6582038d7033.png)

"it's right away after starting. I used the same profile with the latest version. I couldn't start playing as I got stuck right away. I tried as well creating another profile, same issue."
"it's right after you see the intro and you get the first gift"
